src/intTest/resources/fhir/multi_pathology_msg1.fhir.json

```diff
@@ -116,6 +116,35 @@
           "collectedDateTime": "2010-02-23T15:41:00+00:00"
         }
       }
+    },
+    {
+      "fullUrl": "urn:uuid:899d7b43-9e83-4f82-af3a-f0a742acf16f",
+      "resource": {
+        "resourceType": "Observation",
```

Comment:
According to the FHIR documentation, I think "status" is a mandatory field for Observations, but it doesn't seem to be in these test files.